deltacat/tests/utils/data/non_empty_valid.csv

```diff
@@ -0,0 +1,3 @@
+Y	1754-08-30T22:43:41
+""	2022-08-10T22:43:21
+N	2022-08-10T22:43:21
```

Comment:
Should we also test `null` (truly empty values), timestamps that are far in the future, and timestamps with decimal values?

Reply:
Formatter used by DeltaCAT is cleaning it up. Hence, didn't include it, but it pyarrow related functionality.

Reply:
Is that just for nulls? Is that only if the first field/second field is null, or it doesn't matter and the formatter always complains if any field is null? Could we still test timestamps like `9999-08-10T22:43:21.123456`

Reply:
Sure, added year 9999.